recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+    name: pyamg
+    version: "3.0.2"
+
+source:
+    git_url: https://github.com/pyamg/pyamg.git
+    git_tag: v3.0.2
```

Comment:
There's a release here: https://pypi.python.org/pypi/pyamg
How would I reference that?